src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/requests/makerecalldecisions/UpdateRecommendationForNoRecallRequest.kt

```diff
@@ -0,0 +1,233 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.integration.requests.makerecalldecisions
+
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.Status
+
+fun updateRecommendationForNoRecallRequest(status: Status = Status.DRAFT) = """
```

Comment:
I think there is some data in here that isn't relevant to the 'No recall' journey, is it worth updating to keep it clean?